templates/mongo-replicaset/catalogIcon-mongodb.svg

```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
The SVG ends up not fitting in the space allotted.
![screen shot 2015-11-23 at 10 55 49 am](https://cloud.githubusercontent.com/assets/1989084/11344832/c9fc87cc-91d0-11e5-8ab7-faa1e96b6c0c.png)